open_humans/templates/partials/activity-info-table.html

```diff
@@ -50,8 +50,10 @@
     <td>
       <ul>
         <li>
+          <a href="{% url 'members-by-source' %}?source={{ activity.source_name }}">
```

Comment:
It turns out this `get_source_labels_and_names` only includes projects that are acting as data sources... which, to be fair, seems consistent with the API's description.

Maybe it would be nice to change the API's behavior on this, but keeping that the same, we should only render this link on projects when it's judged to be a "data source", consistent with the API – which is based on whether the `returned_data_description` is blank: https://github.com/OpenHumans/open-humans/blob/master/private_sharing/utilities.py#L17